cmds/core/date/date_test.go

```diff
@@ -348,6 +348,12 @@ func TestGetTime(t *testing.T) {
 					t.Errorf("%q failed. Got: %q, Want: %q", tt.name, err, tt.wantErr)
 				}
 			}
+			// Tests which don't dictate the seconds for the date require to be Now().Sencond(). This is flaky, because there is a chance that
+			// if the second is determined in the struct definition tt.wantSec = time.Now().Second() and test execution is a second later.
+			// To avoid this we hardcode the expected second. This is not nice, but an interface would be overkill I think.
+			if tt.wantSec == 0 {
+				tt.wantSec = testTime.Second()
+			}
```

Comment:
if this test is run at 11:59:59, and date runs one second or so later, then date will return 
day + 1, hour 0, minute 0, second 1, but the wantdata might be
dat, hour 11, minute 59, second 0? or do I misunderstand?